Servers/Azure/ARC/instructions.md

```diff
@@ -0,0 +1,49 @@
+# Accessing Azure Account Server From Out of Office
+
+If you are a recognised user on the Cyclecloud and have provided your public key from your Office machine, you will then need to do the following to be able to access the scheduler using your Azure account.
+
+This applies to:
+
+- Out of office
+- Home
+- Docker Container
+
+However, let's first review the steps to allow ARC to access the scheduler and submit jobs.
+
+## File: settings.py
+
+You will need to edit the file `settings.py` and change the following lines:
+
+```python
+
+servers = {
+    'azure': {
+        'cluster_soft': 'Slurm',
+        'address': '{IPADDRESS}', # This line - IP address of the scheduler if it changes
+        'un': '{USERNAME}', # This line - your username
+        'key': '/home/mambauser/.ssh/ubuntu.pem', # This line - path to private key
+        'cpus': 16,
+        'memory': 32,
+        'path': '/mount/nfsshareslurm/nfs/{USERNAME}', # This line - path to shared folder. You will only need to edit the username part of the path
```

Comment:
If we switch `{USERNAME}` to `$USER`, then users won't have to update this

Reply:
They will have to because their username most likely won't be their local computer name. For example, on Azure I use calvinp but on my local computer I use calvin